src/OnboardingSPA/components/Button/NavCardButton/stylesheet.scss

```diff
@@ -1,12 +1,15 @@
 .nfd-nav-card-button {
 	color: var(--nfd-onboarding-white);
 	background-color: var(--wp-admin-theme-color-darker-10);
-	height: 8vh;
-	width: 50vw;
-	margin: 0vw auto;
+	padding: 16px;
+	width: 90%;
```

Comment:
Hey, @abhijitb Maybe we can use 100%? Or we can align the Button in middle? 
I saw that the button is a bit left-aligned due to which it was a bit weird.

Reply:
I used this extension to see the whole spacing https://chrome.google.com/webstore/detail/debug-css/igiofjnckcagmjgdoaakafngegecjnkj

Reply:
Seems pretty well aligned for me :-( 

![welcome-step-alignments](https://user-images.githubusercontent.com/533330/189191691-f20b4726-86df-48cc-ac80-4d7c50b8bde6.PNG)